cg/meta/workflow/microsalt/quality_controller/quality_controller.py

```diff
@@ -85,7 +85,7 @@ def quality_control_sample(self, sample_id: str, metrics: SampleMetrics) -> Samp
         )
```

Comment:
I know this may seem out of scope but from the name of the function is unclear what it does. It also lacks a docstring, it would be great to have one and to rename it as `get_quality_control_sample` if it applies.

Reply:
Do you mean the `quality_control_sample` function?

Reply:
add `perform` at the beginning

Reply:
Added a docstring to clarify what it does 👍